kubernetes_deploy/development/queue_worker_deployment.yml

```diff
@@ -25,6 +25,18 @@ spec:
         imagePullPolicy: Never
         name: worker
         args: ["docker/run_worker.sh"]
+        readinessProbe:
+          exec:
+            command: ["docker/ping.sh"]
```

Comment:
Should this be `docker/ping_services.sh`?

Reply:
😬

Reply:
OK I've finally worked out why this didn't cause the pods to crash out on staging or prod... I completely forgot to add these probes in those environments...

One for tomorrow 😅